### PR TITLE
Download page modifications

### PR DESCRIPTION
**Pull request by sindhuk06**

### CODE REVIEW

**Comment by erickson-doug** [comment]
@sindhuk06 Looks good, but the tabs go to o3de.org -- we can't merge this PR into main and thus the live site until we get those sections populated. @amzn-leenguy to provide more deets

**Comment by amzn-leenguy** [comment]
Hi @sindhuk06 and @erickson-doug  , I may need a preview link to review this PR. Thanks

**Comment by erickson-doug** [comment]
@sptramer Can you help @sindhuk06 fix the DCO signing issue?

**Comment by sptramer** [comment]
@erickson-doug @amzn-leenguy DCO is fixed and ready for any additional review, or can be merged if ready.

**Comment by erickson-doug** [comment]
Preview: https://deploy-preview-725--o3deorg.netlify.app/download/

@sindhuk06 It still defaults to showing the "macOS" as the active tab on page load -- it should be "Windows". The card with the Requirements has the Windows Requirements under macOS, which is also incorrect. We should have separate (blank for now) Requirements cards for each individual OS. Lastly, the tabs for Windows and Linux link to the o3de/o3de GitHub repo -- not a good experience. Please change the following:

- Default to the Windows  ab and the Windows requirements as active when the page is opened..
- Create blank or lorem ipsum cards for macOs and Linux; remove the link to github. I'll fill in the text for those two.

File: assets/sass/custom.sass
Comment (AMZN-daimini) [inline assets/sass/custom.sass:381]: Since it's a clickable button, this should have `cursor: pointer`

File: assets/js/app.js
Comment (AMZN-daimini) [inline assets/js/app.js:118]: nit: remove console.log

File: assets/js/app.js
Comment (AMZN-daimini) [inline assets/js/app.js:120]: This will actually add the "active" class to all parents recursively. I'm not sure it's the intended behavior.

image

**Review by AMZN-daimini — APPROVED** [review]
Think you'll have to fix DCO to submit, but the code looks good to me.

**Comment by sptramer** [comment]
Closing PR in favor of #747

**Pull request closed without merging**
